requirements.txt

```diff
@@ -483,7 +483,7 @@ wcwidth==0.2.5
     # via
     #   prettytable
     #   prompt-toolkit
-websockets==7.0
+websockets==8.1
```

Comment:
`websockets==8.1` is causing conflict

either we upgrade juju to `2.9.4` or pin `websockets < 8.0`

```
The conflict is caused by:
    The user requested websockets==8.1
    juju 2.8.6 depends on websockets<8.0 and >=7.0

To fix this you could try to:
1. loosen the range of package versions you've specified
2. remove package versions to allow pip attempt to solve the dependency conflict
```